doc/protocols/scion-header.rst

```diff
@@ -732,3 +731,255 @@ packet is dropped.
+    |                              HVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+We reduce the size of the MAC field to 2 bytes and assign a 4-byte
+Hop Validation Field (HVF) to the freed space. The total size of the
+Hop Field stays the same (12 bytes).
+The remaining 2 bytes of the MAC are necessary to be able to recompute
+the ``SegID`` when processing a Hop Field against the
+beacon-construction direction:
+:math:`\beta_{i+1} = \beta_i \oplus \sigma_i[:2]`
+(see `Hop Field MAC Computation`_ and `Path Calculation`_)
+
+Therefore, ASes in EPIC-SAPV can still verify path authorization the
+same way it is done for SCION path type packets, even though they
+only contain 2-byte MACs and not 6-byte MACs.
+
+The reason for this modification is to omit blowing up the header
+with additional fields. This is important here because
+one HVF is needed per AS, and hence the header size would increase
+linearly with the number of ASes on the path.
+
+Packet Validation at the Destination Host
+-----------------------------------------
+The destination host validates the authenticity of the packet's
+source and content, and verifies that the packet indeed followed the
+desired path (i.e., the HVFs have all been updated correctly on the
+way). For this, EPIC-SAPV packets also include the SPAO
+option (`SPAO design document`_).
+
+.. _`SPAO design document`: ../protocols/authenticator-option.html
+
+EPIC-SAPV Header Length Calculation
+-----------------------------------
+The length of the EPIC-SAPV path type header is the same as the SCION
+path type header plus 12 bytes (8 bytes for the Packet ID and 4 bytes
+for the OrigPayloadLen and flags).
+
+Procedures
+----------
+
+**Control plane:**
+The beaconing process is the same as for SCION, but the ASes not
+only add the 6 bytes of the truncated MAC to the beacon, but further
+append the remaining 10 bytes (as in EPIC-HP).
+
+**Data plane:**
+The source host fetches the path, including all the 6-byte short hop
+authenticators and the remaining 10 bytes of the authenticators, from
+a (hidden) path server. We refer to the 16 bytes of the fully
+assembled authenticator of the i'th on-path AS as :math:`{ \sigma_i}`.
+Note that the penultimate and the last authenticator of the last
+segment are the same as :math:`{\sigma_{\text{PH}}}` and
+:math:`{\sigma_{\text{LH}}}` in EPIC-HP.
+
+In addition to the authenticators, the source also retrieves the 2nd-level DRKeys
+(:math:`K_i^S` between the source host and the on-path ASes, and
+:math:`K_{SD}` between the source host and the destination host) from
+the certificate server. The source then calculates the Hop Validation
+Fields (:math:`V_i`) and the expected Hop Validation Fields at the
+destination (:math:`U_i`) as follows:
+
+.. math::
+    \begin{align}
+    \text{Origin} &= \text{(SrcISD, SrcAS, SrcHostAddr)} \\
+    W_i &= \text{MAC}_{(K_i^{\text{S}} \oplus \sigma_i)}
```

Comment:
This differs from the EPIC paper. We should at least describe the rationale of this change (and maybe discuss this further, also with Tobias Klenze).